main.go

```diff
@@ -345,6 +348,9 @@ func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *rea
 		args = append(args, rego.AddExternalDataProviderCache(providerCache))
 		mutationOpts.ProviderCache = providerCache
 
+		providerResponseCache := frameworksexternaldata.NewProviderResponseCache(*externaldataProviderResponseCacheTTL, ctx)
```

Comment:
Undefined error? It looks like we have ctx as the last argument of a function. Can we fix this?

Reply:
Undefined is due to the Frameworks PR - https://github.com/open-policy-agent/frameworks/pull/334 We need to merge this first.

Updated the ctx position

Reply:
This is blocked on the frameworks PR you're saying?